WalletWasabi.Fluent/Models/Wallets/CoinModel.cs

```diff
@@ -10,17 +11,36 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 [AutoInterface]
 public partial class CoinModel : ReactiveObject
 {
+	[AutoNotify] private bool _isExcludedFromCoinJoin;
+	[AutoNotify] private int _anonymitySet;
+	[AutoNotify] private bool _confirmed;
+	[AutoNotify] private bool _isCoinJoinInProgress;
+	[AutoNotify] private bool _isBanned;
+	[AutoNotify] private string? _bannedUntilUtcToolTip;
+	[AutoNotify] private string? _confirmedToolTip;
+
 	public CoinModel(Wallet wallet, SmartCoin coin)
 	{
+		Coin = coin;
 		PrivacyLevel = coin.GetPrivacyLevel(wallet.AnonScoreTarget);
 		Amount = coin.Amount;
 		IsConfirmed = coin.Confirmed;
 		Confirmations = coin.GetConfirmations();
 		AnonScore = (int)coin.AnonymitySet;
 		Labels = coin.GetLabels(wallet.AnonScoreTarget);
 		Key = coin.Outpoint.GetHashCode();
+
+		this.WhenAnyValue(c => c.Coin.IsExcludedFromCoinJoin).BindTo(this, x => x.IsExcludedFromCoinJoin);
+		this.WhenAnyValue(c => c.Coin.Confirmed).BindTo(this, x => x.Confirmed);
+		this.WhenAnyValue(c => c.Coin.HdPubKey.AnonymitySet).Subscribe(x => AnonymitySet = (int)x);
+		this.WhenAnyValue(c => c.Coin.CoinJoinInProgress).BindTo(this, x => x.IsCoinJoinInProgress);
+		this.WhenAnyValue(c => c.Coin.IsBanned).BindTo(this, x => x.IsBanned);
+		this.WhenAnyValue(c => c.Coin.BannedUntilUtc).WhereNotNull().Subscribe(x => BannedUntilUtcToolTip = $"Can't participate in coinjoin until: {x:g}");
+		this.WhenAnyValue(c => c.Coin.Height).Select(_ => Coin.GetConfirmations()).Subscribe(x => ConfirmedToolTip = $"{x} confirmation{TextHelpers.AddSIfPlural(x)}");
```

Comment:
Compared to the previous solution, disposing is missing here. Is it not needed anymore?
@ichthus1604 @SuperJMN

Reply:
@soosr My understanding is that `Dispose()` is not needed because of `this.WhenAnyValue()`, which means that the subscription is attached to the current object instance, and not some other instance, thus the lifetime of the subscription is tied to the lifetime of current instance, making it impossible to have leaks.

@SuperJMN is the above statement correct?

Reply:
> @SuperJMN is the above statement correct?

This is correct. Always use this on the left of WhenAnyValue: https://www.reactiveui.net/docs/guidelines/framework/use-this-on-left-of-whenany